packages/Search/src/DTK_LinearBVH_decl.hpp

```diff
@@ -102,12 +102,14 @@ void queryDispatch(
     int const n_results = lastElement( offset );
 
     Kokkos::realloc( indices, n_results );
-    fill( indices, -1 );
+    int const invalid_index = -1;
+    Kokkos::deep_copy( indices, invalid_index );
     if ( distances_ptr )
     {
         Kokkos::View<double *, DeviceType> &distances = *distances_ptr;
         Kokkos::realloc( distances, n_results );
-        fill( distances, Kokkos::ArithTraits<double>::max() );
+        double const invalid_distance = -Kokkos::ArithTraits<double>::max();
```

Comment:
@Rombur Note the minus sign here.  It will be filtered out anyway but I changed it based on our discussion on #321